contrib/docs-code-samples/expand-api-display-access/00-create-tuples/cli.sh

```diff
@@ -18,6 +18,6 @@ files:/photos/beach.jpg#access@(directories:/photos#access)
 files:/photos/mountains.jpg#access@(files:/photos/mountains.jpg#owner)
 files:/photos/mountains.jpg#access@(directories:/photos#access)' | \
   keto relation-tuple parse - --format json | \
-    keto relation-tuple create - >/dev/null \
+    keto relation-tuple create --insecure-disable-transport-security - >/dev/null \
```

Comment:
As these are part of the docs, is there any way we could use TLS, probably through self-signed certs? I assume it will be hard, but just to ensure it is :wink:

Reply:
We could transition to TLS-by-default by using the self-signed approach, and then we would "just" skip hostname verification.

Is this what you had in mind?

Reply:
I would just prefer to not have those insecure flags in the docs samples if possible :sweat_smile: So if there is some way to make that happen it would be highly appreciated, but I guess it will not be as easy.

Reply:
As discussed, we'll update the docs.

Reply:
As discussed, we'll update the docs.